spinner/build.sh

```diff
@@ -48,11 +48,11 @@ function build_service_liferay {
 
 	cp ../../orca/templates/liferay/resources/usr/local/liferay/scripts/pre-startup/10_wait_for_dependencies.sh build/liferay/resources/usr/local/liferay/scripts/pre-startup
 
-	local liferay_image=""
+	local liferay_docker_image=""
```

Comment:
@pedropaccola, there is no ticket prefix (SRE-6066) in the commit message as the previous commit messages, can you please add it?